### ISSUE TITLE
how to scatter points in a validation set?

### ISSUE THREAD

**Issue by hoangson5898** [parrt/dtreeviz#264]
The notebook dtreeviz_sklearn_visualisations.ipynb has shown an example of visualizing two-dimensional feature space by drawing in 3 dimensions, but it has just scattered points in the training set. If I have a validation set, can I plot it?

**Comment by parrt** [comment]
Well our visualizations are talking about how the model is trained so we have no option to show the validation set. sorry.
